charts/opensearch-dashboards/CHANGELOG.md

```diff
@@ -14,6 +14,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 ### Security
 ---
+## [2.26.0]
```

Comment:
Please also update the links at the end of the changelog

Reply:
I see, now updated, I thought it's something automatically generated :D